starlette/authentication.py

```diff
@@ -95,7 +97,7 @@ def sync_wrapper(*args: typing.Any, **kwargs: typing.Any) -> Response:
 
             return sync_wrapper
 
-    return decorator
+    return decorator  # type: ignore
```

Comment:
```suggestion
    return decorator
```

to get the error from mypy